ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -32,17 +32,32 @@ images:
   inputs:
     centos:
       as:
-      - quay.io/app-sre/centos:7
-    openshift_release_golang-1.13:
-      as:
-      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+      - registry.centos.org/centos/centos:7
     rust:
       as:
       - quay.io/app-sre/cincinnati:builder
   to: e2e-test
 promotion:
   name: cincinnati-build-root
   namespace: cincinnati-ci
+raw_steps:
+- pipeline_image_cache_step:
+    commands: |
+      cat <<DOCKERFILE > /opt/Dockerfile
+      FROM centos
+      WORKDIR /go/src/github.com/openshift/cincinnati/
+      COPY . .
+      DOCKERFILE
+    from: centos
+    to: src-dockerfile
```

Comment:
I wonder if we create the Dockerfile here or we should add the source docker file to the Cincinnati repository.

Reply:
The safest way is to go with `src-dockerfile`, update `Dockerfile.build` in Cincinnati later and remove `src-dockerfile`. This way we'll know if things broke down